tests/legacy-cli/e2e/tests/build/multiple-configs.ts

```diff
@@ -65,7 +72,7 @@ export default async function () {
   await expectFileToExist('dist/test-project/main.js.map');
   await expectToFail(() => expectFileToExist('dist/test-project/vendor.js'));
   // Use three configurations and a override, and prod at the end.
-  await ng('build', '--configuration=one,two,three', '--vendor-chunk=false', '--prod');
+  await ng('build', '--configuration=one,two,three', '--vendor-chunk=false');
```

Comment:
The comment above should probably be changed as well since `--prod` is not at the end anymore.